app/Models/Campaign.php

```diff
@@ -267,4 +269,14 @@ public function setArbitraryDateString($attribute, $value)
 
         $this->attributes[$attribute] = $this->fromDateTime($value);
     }
+
+    /**
+     * Get the indexable data array for the model.
+     *
+     * @return array
+     */
+    public function toSearchableArray()
+    {
+        return Arr::except($this->toArray(), ['impact_doc']);
```

Comment:
This is pretty nice. We can easily _remove_ items to be indexed by passing them into this array. `impact_doc` for example is attribute we don't need indexed by Algolia so we remove it before sending off the data to the Search API.

Reply:
Super neat!